guides/common/modules/con_performance-tuning-quickstart.adoc

```diff
@@ -26,5 +26,6 @@ The default value is `default`.
 ----
 # {foreman-installer} --tuning "_My_Installation_Size_"
 ----
+. Optional: There are some additional checks you can do in <<Applying configurations_{context}>> section.
```

Comment:
Please use `xref:_My_Anchor_{context}[_My_Title_]`.

We generally say "For more information, see xref:...."

Reply:
Should be fixed, thank you